requirements.txt

```diff
@@ -47,6 +47,7 @@ black==22.12.0
 testfixtures==6.18.5
 pytest==7.1.1
 pytest-datadir==1.3.0
+pytest-mock>=3.0.0
```

Comment:
Is this used in the tests? I see putest and requests-mock being imported but not pytest-mock

Reply:
It creates a pytest fixture called "mocker", which is used in the pytest fixtures in[ this file](https://github.com/austinweisgrau/parsons/blob/match/test/test_catalist/conftest.py)